misk-schema-migrator-gradle-plugin/src/test/resources/schema-migrator-plugin-test/src/main/resources/db-migrations/v0001__create-people-table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE people (
+    id                          bigint       NOT NULL AUTO_INCREMENT,
```

Comment:
Not sure the tabbed format is necessary or aligns with what is usual input to schema migrator

Reply:
I copied from the first example I saw but may not be idiomatic, I'll change.